doc/book/controllers.md

```diff
@@ -279,7 +279,12 @@ extending class.
 
 ## AbstractConsoleController
 
-`Zend\Mvc\Controller\AbstractConsoleController` extends from [AbstractActionController](#abstractactioncontroller)
+> To use  it, you need to install zend-mvc-console via
+```bash
+composer require zendframework/zend-mvc-console --sort-packages
```

Comment:
Is the option `--sort-packages` really needed in this documentation. I think no.